src/core/lib/security/credentials/external/external_account_credentials.cc

```diff
@@ -273,6 +273,20 @@ std::string ExternalAccountCredentials::debug_string() {
                          grpc_oauth2_token_fetcher_credentials::debug_string());
 }
 
+std::string ExternalAccountCredentials::MetricsHeaderValue() {
+  return absl::StrFormat("gl-cpp/%s auth/%s google-byoid-sdk source/%s sa-impersonation/%v config-lifetime/%v",
+      "unknown",
```

Comment:
This string constant should just be inlined into the format string rather than being substituted in at runtime.